lib/popcorn/pop_auth.nit

```diff
@@ -80,6 +80,15 @@ module pop_auth
 import pop_handlers
 import github
 
+# Initialize session in request if non existent.
+#
+# Should be called before any use of the session.
+class SessionInit
+	super Handler
+
+	redef fun all(req, res) do if req.session == null then req.session = new Session
```

Comment:
`pop_auth` requites the importation of github, isn't that too much?
One might assume that `SessionInit` is light and useful enough to be provided by default